Tasks/AzureFileCopy/AzureFileCopyJob.ps1

```diff
@@ -43,10 +44,13 @@ param (
     }
     else
     {
-        if(Test-Path "$env:AGENT_HOMEDIRECTORY\externals\vstshost")
-        {
-            [void][reflection.assembly]::LoadFrom("$env:AGENT_HOMEDIRECTORY\externals\vstshost\Microsoft.TeamFoundation.DistributedTask.Task.LegacySDK.dll")
-        }
+        #if(Test-Path "$env:AGENT_HOMEDIRECTORY\externals\vstshost")
+        #{
+            #[void][reflection.assembly]::LoadFrom("$env:AGENT_HOMEDIRECTORY\externals\vstshost\Microsoft.TeamFoundation.DistributedTask.Task.LegacySDK.dll")
+        #}
+
+        Import-Module "$deploymentUtilitiesLocation\Microsoft.TeamFoundation.DistributedTask.Task.Deployment.Internal"
+        Import-Module "$deploymentUtilitiesLocation\Microsoft.TeamFoundation.DistributedTask.Task.DevTestLabs.dll"
```

Comment:
We are importing these which were already imported in parent script. a) Is it required? b) If so, does it apply to vstsAzureHelpers_?